### PR TITLE
Add debug messages for availability_message

### PR DESCRIPTION
**Pull request by hsong-rh**

### CODE REVIEW

File: config/initializers/clowder_config.rb [hunk 1]
```diff
@@ -65,6 +65,8 @@ def self.instance
 
       # TODO: update with valid url later
       options["CLOUD_CONNECTOR_URL"] = ENV["CLOUD_CONNECTOR_URL"] 
+      options["SOURCES_URL"] = ENV["SOURCES_URL"]
```
Comment (mkanoor) [inline config/initializers/clowder_config.rb:68]: @hsong-rh Is this also needed or can you make a different PR for that.

Comment (hsong-rh) [reply config/initializers/clowder_config.rb:68]: I can add this in another PR

**Review by mkanoor — APPROVED** [review]
